lib/lib.es5.d.ts

```diff
@@ -22,8 +22,8 @@ and limitations under the License.
 /// ECMAScript APIs
 /////////////////////////////
 
-declare const NaN: number;
-declare const Infinity: number;
+declare var NaN: number;
```

Comment:
I am just wondering on the reason for changing const to var in .d.ts files ? @weswigham